whitepaper/chapter01.md

```diff
@@ -0,0 +1,150 @@
+# 1. Cloud Native Thinking for Telecommunications
+
+Basic tenets and end users suggestions on how cloud native design and principles can be applied to mission critical telecommunications functions.
+
+## Table of Contents
+* [1.1 Introduction](#1.1)
+* [1.2 What is Cloud Native?](#1.2)
+* [1.3 Defining Cloud Native Systems](#1.3)
+  * [1.3.1 What "loosely coupled" means in cloud native systems](#1.3.1)
+  * [1.3.2 Cloud native applications require orchestration](#1.3.2)
+  * [1.3.3 Infrastructure, Deployment and Configuration of Cloud Native Systems](#1.3.3)
+* [1.4 Cloud Native Network Functions](#1.4)
+* [1.5 Cloud Native for Telcos](#1.5)
+  * [1.5.1 A Brief History of Virtualisation for Telcos](#1.5.1)
+  * [1.5.2 Software Defined Networking And The Emergence of VNFs](#1.5.2)
+  * [1.5.3 Principles for Cloud Native Telco Infrastructure and Applications](#1.5.3)
+  * [1.5.4 Evolving the Stack From VNFs to CNFs](#1.5.4)
+* [1.6 Cloud Native for Telcos in Practice](#1.6)
+* [1.7 Conclusion](#1.7)
+
+<a name="1.1"></a>
+## 1.1 Introduction
+
+Kubernetes or containers for workloads has celebrated its 5th Birthday recently , during this time all industry has seen wider support and adoption of this technology to build the infrastructure . It is logical that operators worldwide who have already embarked on an ambitious journey to transformation show interest in to it as a catalyst to support transformation . It will not only makes a perfect business senses to do so but it will enable networks and applications to be treated alike IT or web applications , a promise if fulfilled will support rapid digitisation and cloudification of networks. This is why understanding cloud native journey from an operator’s community perspective is so important.
```

Comment:
The wording needs updating in this section.  Referencing K8s and it's birthday are probably not needed.  Focus on cloud native introduction to operators.

Reply:
My suggestion is we revert back to the original introduction with Bell Canada. It is a nice anecdote and frames the story. As follows:

If you use voicemail on a wireless network powered by Bell Canada, your message queue is officially cloud native. In 2019, Bell Canada ported its voicemail network functions over to a container-based architecture running on top of a Kubernetes orchestration layer alongside its production instance of the ONAP architecture. Voicemail is one of a number of functions that Bell Canada has or will be migrating to Cloud Native Network Functions (CNFs). “The main driver behind cloud native for Bell Canada is service availability, namely, how can we leverage cloud native principles to make sure our services are more agile, more flexible, and more resilient?,” explains Daniel Bernier, a Senior Technical Architect at Bell Canada and a member of Cloud Native Computing Foundation’s (CNCF) Telecom User Group (TUG). 

Telecommunications companies provide some of the most critical services in the world. From emergency 911 services to air traffic control to buy-sell signals that move billions of dollars worth of bonds, telecommunications networks impact our safety, social stability, and economic prosperity. s. Because the stakes are so high, telecommunications infrastructure has been designed, maintained, and upgraded to minimize risks and maximize reliability and uptime. 

Operators like Bell Canada intend to move more of their functionality to cloud native architectures as the telecommunications community further evolves standards and practices around deployment of mission critical applications and services over CNFs. 

The goal of this whitepaper is to provide an introduction to cloud native principles as they might apply to telecom operators and enterprises running internal telecommunications-like infrastructure.  By helping operators and others in the industry to better understand what cloud native means for telecommunications, we can help build consensus around industry adoption of cloud native technologies and all enjoy the benefits of this transition.